core/src/main/scala/org/apache/spark/sql/execution/joins/LocalJoin.scala

```diff
@@ -411,14 +409,10 @@ case class LocalJoin(leftKeys: Seq[Expression],
     }
     val streamKeyVars = ctx.generateExpressions(streamKeys)
 
-    val mapAccesCode = mapAccessor.generateMapLookup(entryVar, localValueVar, keyIsUniqueTerm,
+    mapAccessor.generateMapLookup(entryVar, localValueVar, keyIsUniqueTerm,
```

Comment:
Yes, just pushed a fix whereby the checkCondition's code is separately emitted with rest of the inputCodes at the top.